### ISSUE TITLE
Confusing log message

### ISSUE THREAD

**Issue by Spritekin** [krpn/prometheus-alert-webhooker#28]
I have noticed an error message in the log is misleading so it sent me to a wrong direction until I noticed the problem. Let me explain.

Following the examples I had configured my rule with a block of 10m, then setup some shell rules.

I then submitted some alerts for testing but noticed only my first alert was executed, any alert after the first one reported as failed with this message:

```
{"alert":"jenkins-job-failed","context":"runner","details":{"command":"sh"},"duration":"3.413µs","event_id":"273f","executor":"shell","level":"debug","msg":"runner got executing task #1/1 unsuccessful result, stopping group: in_block","result":"in_block","rule":"onJenkinsJobAlert","tasks":[{"alert":"jenkins-job-failed","details":{"command":"sh"},"event_id":"273f","executor":"shell","rule":"onJenkinsJobAlert"}],"time":"2021-09-02T13:28:13Z"}
```

So I got really confused on why the first call succeeded and the others reported "unsuccessful result"? I spent long hours trying to understand it, went into the container and run the command manually with success so I couldn't understand the reason why it kept saying "unsuccessful result".

So after a while and reading the docs, I understand the reason it is blocked is because the execution depends on the "block" parameter which would unblock after 10 mins. That is not really clear after reading the docs.

However my greatest confusion was that I left it running overnight while I was thinking about it then when I opened the next morning I got flooded with the firing of all my tests, meaning it wasn't really "unsuccessful" and dropped, it just keeps the event on hold for the specified length of time then it processes the next task.

So my suggestion is:
a. Improve the description of the "block" parameter in the documentation.
b. Change the log text on a blocked execution event to something more descriptive like:
"The event was scheduled for execution but the runner is still blocked. The event will be put on hold until the runner is unblocked (change the rule block parameter to adjust this behaviour)"

Regards,

**Comment by krpn** [comment]
I have added `Understanding blocking` section in `README.md`.
